site/_docs/reference.md

```diff
@@ -567,6 +567,7 @@ JSON,
 **JSON_OBJECT**,
 **JSON_OBJECTAGG**,
 **JSON_QUERY**,
+**JSON_TYPE**,
```

Comment:
Is `JSON_TYPE` currently a reserved keyword of MySQL?

And you may want to document usage of `JSON_TYPE` also. Something like: http://calcite.apache.org/docs/reference.html#query-functions

Reply:
I recall that something like \*\*JSON_TYPE\*\* indicates a reserved keyword.

If you don't want make a keyword reserved, maybe you should add it to https://github.com/apache/calcite/blob/master/core/src/main/codegen/templates/Parser.jj#L6690

Reply:
I've added non-reserved keywords

Reply:
Could you please add non-reserved keyword `JSON_TYPE` to here or nearby:

https://github.com/apache/calcite/blob/d9fe7f7ad7c2006780e459ce90e3f442a6246051/core/src/main/codegen/config.fmpp#L161

Reply:
hi @zhztheplayer My attempts to use non-reserved keywords resulted in many test conflicts and too many changes. I'll keep the keywords
best
qianjin